hack/e2e.sh

```diff
@@ -102,5 +102,19 @@ while [ $ATTEMPTS -ge 0 ]; do
   fi
 done
 
+# Reconfigure monitoring operator to support user workloads
+# https://docs.openshift.com/container-platform/4.3/monitoring/monitoring-your-own-services.html
+cat > /tmp/config.yaml << EOF
+techPreviewUserWorkload:
+  enabled: true
+EOF
+oc -n openshift-monitoring create configmap cluster-monitoring-config --from-file=/tmp/config.yaml
```

Comment:
```suggestion
oc -n openshift-monitoring create configmap cluster-monitoring-config \
  --from-file=<(cat <<'EOF'
  techPreviewUserWorkload:
    enabled: true
EOF
)
```

Reply:
I don't like the fact that we create a file in `/tmp`, but I think I don't like bash version more :) Its really tough to debug these kind of issues.

Do you mind if I keep the tmp file version?

Reply:
Trevor has suggest the most elegant way